spec/unit/util/run_mode_spec.rb

```diff
@@ -10,14 +10,18 @@
     expect(@run_mode.run_dir).to eq('$vardir/run')
   end
 
+  it "has logdir depend on vardir" do
+    expect(@run_mode.log_dir).to eq('$vardir/log')
```

Comment:
Is this accurate?  Judging from some of the other comments about rundir and logdir it sounds like we no longer expect this to be the case.  The spec is testing the implementation of RunMode#log_dir, but at runtime either UnixRunMode or WindowsRunMode is used, and they both override the implementation of RunMode#log_dir so it's not under vardir.

Maybe I'm too deep in the weeds and confusing myself, but it feels like we have some inconsistencies?